tasks/README.md

```diff
@@ -198,4 +198,7 @@ Name | Summary | Category
 `task205_remove_even_elements.json` | Given a list of integers remove all elements that are even | Answer Generation, Arithmetic
 `task206_collatz_conjecture.json` | Given a list of integers compute the next number in the *3n+1* problem | Answer Generation, Arithmetic
 `task207_max_element_lists.json` | Given a list of lists of integers compute the max value for each list | Answer Generation, Arithmetic
-`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
\ No newline at end of file
+`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
+`task246_dream_question_generation` | Writing a question based on a given conversation | Question Generation
```

Comment:
Maybe change to:
`Given a conversation, generate a multiple-choice question based on it.`

Reply:
Sure!